docs/book/src/getting-started.md

```diff
@@ -125,10 +125,10 @@ Now, see our example fully completed.
 
 #### Generating manifests with the specs and validations
 
-To generate the required CRDs we will run `make generate` command, which will call [controller-gen][controller-gen]
+To generate the required CRDs we will run `make manifests` command, which will call [controller-gen][controller-gen]
```

Comment:
Here we need the macke generate right?
Since we changed the spec

Reply:
Thanks for the quick response!
make generate doesn't seem to do what this is saying this will do... Sorry very new to kubebuilder so I could be doing something wrong here:

```
(.venv) (devbox) [trent@Krum:~/Source/kubebuilder-demo/memcached-operator] [master] $ tree config/crd/
config/crd/
├── kustomization.yaml
└── kustomizeconfig.yaml

1 directory, 2 files
(.venv) (devbox) [trent@Krum:~/Source/kubebuilder-demo/memcached-operator] [master] $ make generate
/home/trent/Source/kubebuilder-demo/memcached-operator/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
(.venv) (devbox) [trent@Krum:~/Source/kubebuilder-demo/memcached-operator] [master] $ tree config/crd/
config/crd/
├── kustomization.yaml
└── kustomizeconfig.yaml

1 directory, 2 files
(.venv) (devbox) [trent@Krum:~/Source/kubebuilder-demo/memcached-operator] [master] $ make manifests
/home/trent/Source/kubebuilder-demo/memcached-operator/bin/controller-gen rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
(.venv) (devbox) [trent@Krum:~/Source/kubebuilder-demo/memcached-operator] [master] $ tree config/crd/
config/crd/
├── bases
│   └── cache.example.com_memcacheds.yaml
├── kustomization.yaml
└── kustomizeconfig.yaml

2 directories, 3 files
```

just following along with the docs and running into this issue

Reply:
ohh I see what you're saying - we need both commands I expect. that's a great point! let me see if I can come up with something more clean than `make generate && make manifests`

Reply:
make generate && make manifests
Taht seems great !!!

Reply:
To generate all required files, run `make generate` to create the DeepCopy implementations in `api/v1alpha1/zz_generated.deepcopy.go`, followed by `make manifests` to generate the CRD manifests under `config/crd/bases`. Both commands use [controller-gen][controller-gen] with different flags for code and manifest generation respectively.

does that wording sound OK? just want to make sure i'm not missing anything critical before making another commit - updating. Thanks for your help! I haven't read too far ahead in the docs yet so i'm not sure how verbose vs concise the wording should be!

Reply:
Yes 
I loved. Just a tiny refinement :-) 

```md
To generate all required files:

1. Run `make generate` to create the DeepCopy implementations in `api/v1alpha1/zz_generated.deepcopy.go`.

2. Then, run `make manifests` to generate the CRD manifests under `config/crd/bases` and a sample for it under `config/crd/samples`.

Both commands use [controller-gen][controller-gen] with different flags for code and manifest generation, respectively.
```

WDYT?

Reply:
ohhh that is a wonderful callout in reguards to the sample dir as well!

That is perfect - thanks a bunch for the help, I've commited the changes!